web/src/engine/websites/ComicK_e2e.ts

```diff
@@ -17,8 +17,8 @@ const config: Config = {
     },
     entry: {
         index: 2,
-        size: 854_209,
-        type: 'image/png'
+        size: 93_076,
```

Comment:
Is this still the correct image? Change in size seems a bit suspicious.

Reply:
I've double checked the chapter, its fine 🤷

Reply:
Apparently, tachiyomi parameter in url give compressed pictures :D. Good thing we removed it.